### PR TITLE
Add/FIX games

### PR DESCRIPTION
**Pull request by n8225**

### CODE REVIEW

**Comment by Kickball** [comment]
@AndyR207 @nodiscc a major overhaul of the games section, all changes and additions look good to me.

Untrusted's description doesn't explain very well exactly what the game is about/its mechanics, but I feel like I am nitpicking a bit here.

File: README.md [hunk 1]
```diff
@@ -547,10 +547,15 @@ Some [Groupware](#groupware) solutions also feature file sharing and synchroniza
 
 ## Games
 
-  * [20euros](https://github.com/jatekos101/20euros) - Clone of 2048 game with euros. - Unlicensed
-  * [Agar.IO Clone](https://github.com/huytd/agar.io-clone) - Agar.io clone written with Socket.IO and HTML5 canvas - `MIT`
-  * [Cubiks-2048](https://github.com/Kshitij-Banerjee/Cubiks-2048) - Clone of 2048 game in 3D. - `CCANC 4.0`
+  * [0hh1](https://github.com/Q42/0hh1) - A lovely little logic game by Q42. ([Demo](http://0hh1.com/)) `MIT` `HTML5`
+  * [20euros](https://github.com/jatekos101/20euros) - Clone of 2048 game with euros. ([Demo](http://jatekos101.github.io/20euros/)) `Unlicensed` `HTML5`
+  * [Agar.IO Clone](https://github.com/huytd/agar.io-clone) - Agar.io clone written with Socket.IO and HTML5 canvas `MIT` `Nodejs`
```
Comment (AndyR207) [inline README.md]: Needs a full stop after the descriptions.

File: README.md [hunk 2]
```diff
@@ -547,10 +547,15 @@ Some [Groupware](#groupware) solutions also feature file sharing and synchroniza
...
+  * [Clumsy Bird](https://github.com/ellisonleao/clumsy-bird) - A MelonJS port of the famous Flappy Bird Game. ([Demo](http://www.ellison.rocks/clumsy-bird/)) `MIT` `Nodejs`
+  * [Cubiks-2048](https://github.com/Kshitij-Banerjee/Cubiks-2048) - Clone of 2048 game in 3D. ([Demo](https://kshitij-banerjee.github.io/Cubiks-2048/)) `CCANCv4` `HTML5`
+  * [A Dark Room](https://github.com/doublespeakgames/adarkroom) - A minimalist text adventure game for your browser. ([Demo](http://adarkroom.doublespeakgames.com/)) `MPLv2` `HTML5`
+  * [Hextris](https://github.com/Hextris/hextris) - Fast paced HTML5 puzzle game inspired by Tetris! ([Demo](https://github.com/Hextris/hextris)) `GPLv3` `HTML5`
```
Comment (AndyR207) [inline README.md]: Replace the exclamation mark with a full stop please.

File: README.md [hunk 3]
```diff
@@ -547,10 +547,15 @@ Some [Groupware](#groupware) solutions also feature file sharing and synchroniza
...
-  * [Agar.IO Clone](https://github.com/huytd/agar.io-clone) - Agar.io clone written with Socket.IO and HTML5 canvas - `MIT`
-  * [Cubiks-2048](https://github.com/Kshitij-Banerjee/Cubiks-2048) - Clone of 2048 game in 3D. - `CCANC 4.0`
+  * [0hh1](https://github.com/Q42/0hh1) - A lovely little logic game by Q42. ([Demo](http://0hh1.com/)) `MIT` `HTML5`
+  * [20euros](https://github.com/jatekos101/20euros) - Clone of 2048 game with euros. ([Demo](http://jatekos101.github.io/20euros/)) `Unlicensed` `HTML5`
+  * [Agar.IO Clone](https://github.com/huytd/agar.io-clone) - Agar.io clone written with Socket.IO and HTML5 canvas `MIT` `Nodejs`
+  * [Clumsy Bird](https://github.com/ellisonleao/clumsy-bird) - A MelonJS port of the famous Flappy Bird Game. ([Demo](http://www.ellison.rocks/clumsy-bird/)) `MIT` `Nodejs`
+  * [Cubiks-2048](https://github.com/Kshitij-Banerjee/Cubiks-2048) - Clone of 2048 game in 3D. ([Demo](https://kshitij-banerjee.github.io/Cubiks-2048/)) `CCANCv4` `HTML5`
+  * [A Dark Room](https://github.com/doublespeakgames/adarkroom) - A minimalist text adventure game for your browser. ([Demo](http://adarkroom.doublespeakgames.com/)) `MPLv2` `HTML5`
```
Comment (AndyR207) [inline README.md]: This should be in the 'A' section, not the 'C' section.

**Comment by n8225** [comment]
Any other issues?

**Comment by nodiscc** [comment]
@n8225 please replace all occurrences of `CCA...` with `CCBY...` since it's the most commonly used acronym.

**Pull request merged**